core/src/main/java/org/elasticsearch/index/engine/EngineConfig.java

```diff
@@ -89,7 +89,15 @@
         }
     }, Property.IndexScope, Property.NodeScope);
 
-    private TranslogConfig translogConfig;
+    /**
+     * Configures an index to optimize documents with auto generated ids for append only. If this setting is updated to <code>false</code>
+     * it might not take effect immediately. Depending on the engine implementation a change to this setting won't be reflected until the
```

Comment:
question: I think going from true to false is effective immediately, it's going from false to true that's problematic, right?